app/samples/tftp_server/include/c_receiver.h

```diff
@@ -0,0 +1,34 @@
+#ifndef HELLOTENANT_H
```

Comment:
do you just make up include guard names lol? I thought we kinda standardized on doing it by file name